scripts/requests/README.md

```diff
@@ -0,0 +1,21 @@
+# HTTP Request Collection
+
+This directory contains a collection of HTTP requests primarily intended for development and testing of the ORT Server API. 
+These examples showcase common operations and can be useful when working with the API during development.
+The collection is not exhaustive but provides examples for common operations.
+For the complete Server API reference, please see [the API documentation](https://eclipse-apoapsis.github.io/ort-server/api/ort-server-api).
+
+## Environments
+
+The HTTP requests use variables defined in environment files:
+
+- Base configuration: [http-client.env.json](env/http-client.env.json)
+- Custom configuration: Create a file called `http-client.private.env.json` to override the default values with your own settings
+
+## Clients
+
+The request format in this collection is compatible with several HTTP clients, including:
```

Comment:
Are they also compatible with any of Postman, Insomnia, https://github.com/usebruno/bruno, https://github.com/mountain-loop/yaak, or https://github.com/alexandrehtrb/Pororoca, and should we list these as well (as I believe these are more common than those neovim extensions)?

Reply:
> Are they also compatible with any of Postman, Insomnia, https://github.com/usebruno/bruno, https://github.com/mountain-loop/yaak, or https://github.com/alexandrehtrb/Pororoca, 

Sadly, these tools are not compatible with this format. See e.g. https://github.com/usebruno/bruno/issues/1537.

> as I believe these are more common than those neovim extensions

Listing every tool that supports this format [1] was one of my main motivations to add this documentation, because unfortunately, JetBrains limits this feature to Ultimate users.
So ORT server developers not using IntelliJ Ultimate can still use this collection.

[1]: Which I really like, compared to unreviewable, undocumented XML files.

Reply:
> Sadly, these tools are not compatible with this format. See e.g. [usebruno/bruno#1537](https://github.com/usebruno/bruno/issues/1537).

Thanks, upvoted!